src/utils/api/constants.ts

```diff
@@ -1 +1 @@
-export const BASEURL = "http://nodes.jup.io/nxt?";
+export const BASEURL = "http://localhost:3080/nxt?"; // proxy runs on 3080 by default
```

Comment:
make this configured as the localhost proxy based on the DEV or PROD build status. this may involve setting a global like APP_VERSION or vite may have a VITE_ variable already available to utilize as a global.